opentype-shaping-devanagari.md

```diff
@@ -1192,6 +1214,22 @@ conjunct ligatures. These sequences must match "_Consonant_,Halant,_Consonant_".
 A sequence matching "_Consonant_,Halant,ZWJ,_Consonant_" or
 "_Consonant_,Halant,ZWNJ,_Consonant_" must not be flagged to form a conjunct.
 
+> Note: The presence of the "ZWJ" at the end of the sequence means
+> that the sequence will match the regular-expression test in stage 1
```

Comment:
If my recollection of "C, H, ZWJ, C" being one syllable is found to be correct, we should change "will match" to "can match", or consider rewording entirely.